src/Engine/Renderer/Camera/Camera.hpp

```diff
@@ -153,9 +148,11 @@ class RA_ENGINE_API Camera final {
 
     Scalar m_width;  // Viewport width (in pixels)
     Scalar m_height; // Viewport height (in pixels)
-    Scalar m_aspect;
+    Scalar m_aspect; // FIXME: never used.

```

Comment:
Nothing to fix here, aspect ratio ha no utility for camera management. (see comments above)